src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorToKfsVendorDetailConversionServiceImpl.java

```diff
@@ -390,6 +525,16 @@ private boolean isNotPmwVendorIndividualSolePropriatorSingleMemberLlc(PaymentWor
                !(pmwVendor.getRequestingCompanyTaxClassificationCode().intValue() == PaymentWorksConstants.INDIVIDUAL_SOLE_PROPRIETOR_TAX_CLASSIFICATION_INDICATOR)) ? true : false;
     }
     
+    private boolean isCanadaFipsCountryCode (String countryCode) {
+        return (StringUtils.isNotBlank(countryCode) &&
+                StringUtils.equalsIgnoreCase(countryCode, PaymentWorksConstants.COUNTRY_CODE_CANADA));
```

Comment:
StringUtils.equalsIgnoreCase() should already be null-safe, so you shouldn't need a non-blank check here.

Reply:
Changing as requested.